source/clear-linux/tutorials/aws-web/aws-web.rst

```diff
@@ -0,0 +1,266 @@
+:abbr:`AWS (Amazon Web Services)` console and complete the following tasks:
+
+#. Locate and select the |CL| OS Basic AMI in the AWS Marketplace.
+#. Create a new public and private key pair to allow you to connect to your
+   |CL| instance securely.
+#. Launch the new |CL| instance and connect to it.
+#. Update your instance of |CL| using the :command:`swupd` command.
+#. Stop the |CL| instance.
+
+Prerequisites
+*************
+
+This tutorial assumes the following:
+* You are using a linux-based system to access AWS and can run :command:`SSH`
+  to access the remote |CL| AWS image.
+* Your browser puts downloaded files in the :file:`$HOME/Downloads` directory.
+* You have already set up an AWS user account and logged in to the AWS
+  console.
+  
+.. note::
+   This tutorial uses a |CL| AMI t2.micro instance that is eligible for the
+   AWS free tier. To learn more about AWS and setting up an account, visit the
+   AWS website at http://aws.amazon.com.
+
+Locate, select, and launch the Clear Linux OS Basic AMI
+*******************************************************
+
+#. Start from your main AWS console, select :guilabel:`EC2` as shown in
+   figure 1:
+
+   .. figure:: figures/aws-web-1.png
+      :scale: 50 %
+      :alt: AWS Console
+
+      Figure 1: :guilabel:`AWS Console`
+
+   This brings up your :guilabel:`EC2 Dashboard`.
+
+#. To create a new |CL| instance from the :guilabel:`EC2 Dashboard` menu,
+   select the :guilabel:`Launch Instance` as shown in figure 2:
+
+   .. figure:: figures/aws-web-2.png
+      :scale: 50 %
+      :alt: EC2 Dashboard
+
+      Figure 2: :guilabel:`EC2 Dashboard`
+
+   This takes you to the
+   :guilabel:`Step 1: Choose an Amazon Machine Image (AMI)` menu.
+
+#. To find the :guilabel:`|CL| OS Basic` AMI in the
+   :guilabel:`Step 1: Choose an Amazon Machine Image (AMI)` menu, do the
+   following:
+   
+   #. In the lefthand navigation window, select the
+      :guilabel:`AWS Marketplace` to bring up the search bar to
+      :guilabel:`Search AWS Marketplace Products`.
+
+   #. In the search bar, type `clear linux os` and hit the :kbd:`Enter` key to
+      search for and locate the :guilabel:`|CL| OS Basic` AMI.
+
+   #. Select the :guilabel:`|CL| OS Basic` AMI by clicking on the
+      :guilabel:`Select` button as shown in figure 3:
+
+      .. figure:: figures/aws-web-3.png
+         :scale: 50 %
+         :alt: Step 1: Choose AMI
+
+         Figure 3: :guilabel:`Step 1: Choose AMI`
+
+   #. A pop-up dialog box appears showing you more information about the
+      :guilabel:`|CL| OS Basic` AMI along with the pricing details for running
+      |CL| on different platform configurations as shown in figure 4. Select
+      the :guilabel:`Continue` button.
+
+      .. figure:: figures/aws-web-4.png
+         :scale: 50 %
+         :alt: Clear Linux OS Basic
+
+         Figure 4: :guilabel:`Clear Linux OS Basic`
+
+#. The :guilabel:`Choose Instance Type` menu appears as shown in figure 5.
+
```

Comment:
Move figure 5 right after it is mentioned in the text. The explanatory paragraph and note can come afterwards.

Reply:
done